src/tests/functional/plugin/shared/src/snippets/transpose.cpp

```diff
@@ -12,14 +12,18 @@ namespace test {
 namespace snippets {
 
 std::string Transpose::getTestCaseName(testing::TestParamInfo<ov::test::snippets::TransposeParams> obj) {
-    ov::PartialShape inputShape;
+    InputShape inputShapes;
     std::vector<int> order;
     std::string targetDevice;
     size_t num_nodes, num_subgraphs;
-    std::tie(inputShape, order, num_nodes, num_subgraphs, targetDevice) = obj.param;
+    std::tie(inputShapes, order, num_nodes, num_subgraphs, targetDevice) = obj.param;
 
     std::ostringstream result;
-    result << "IS=" << ov::test::utils::partialShape2str({inputShape}) << "_";
+    result << "IS=" << ov::test::utils::partialShape2str({inputShapes.first}) << "_";
+    result << "TS=";
+    for (const auto& shape : inputShapes.second) {
+        result << "(" << ov::test::utils::vec2str(shape) << ")_";
+    }
```

Comment:
Minor: `operator<<` [supports](https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp#L18) `InputShape`, so it seems like we can just use `result << "IS=" << inputShapes << "_"`